circup.py

```diff
@@ -804,3 +804,7 @@ def install(name, py):  # pragma: no cover
         click.echo("Installed '{}'.".format(name))
     else:
         click.echo("Unknown module named, '{}'.".format(name))
+
+# Allows execution via `python -m circup ...`
+if __name__ == "__main__":
```

Comment:
Please change to:

```
if __name__ == "__main__":  # pragma: no cover
```

... to appease the code coverage reporting gods. :wink: (try `make check` to see what I mean).